src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/config.html.twig

```diff
@@ -1,115 +1,103 @@
-            <span>
-                {% if collector.applicationname %}
-                    {{ collector.applicationname }} {{ collector.applicationversion }}
-                {% elseif collector.symfonyState is defined %}
-                    {% if 'unknown' == collector.symfonyState -%}
-                    <span class="sf-toolbar-status sf-toolbar-info-piece-additional" title="Unable to retrieve information about the Symfony version.">
-                    {%- elseif 'eol' == collector.symfonyState -%}
-                    <span class="sf-toolbar-status sf-toolbar-status-red" title="This Symfony version will no longer receive security fixes.">
-                    {%- elseif 'eom' == collector.symfonyState -%}
-                    <span class="sf-toolbar-status sf-toolbar-status-yellow" title="This Symfony version will only receive security fixes.">
-                    {%- elseif 'dev' == collector.symfonyState -%}
-                    <span class="sf-toolbar-status sf-toolbar-status-yellow" title="This Symfony version is still in the development phase.">
-                    {%- else -%}
-                    <span class="sf-toolbar-status sf-toolbar-status-green">
-                    {%- endif -%}
-                    {{ collector.symfonyversion }}</span>
-                {% endif %}
+        {% if collector.symfonyState is defined %}
+            <span class="sf-toolbar-label">
+                {{ include('@WebProfiler/Icon/symfony.svg.twig') }}
             </span>
-        </a>
+            <span class="sf-toolbar-value">{{ collector.symfonyversion }}</span>
+        {% elseif collector.applicationname %}
+            <span class="sf-toolbar-label">{{ collector.applicationname }}</span>
+            <span class="sf-toolbar-value">{{ collector.applicationversion }}</span>
+        {% endif %}
     {% endset %}
+
     {% set text %}
         {% if collector.applicationname %}
             <div class="sf-toolbar-info-piece">
                 {{ collector.applicationname }} <b>{{ collector.applicationversion }}</b>
             </div>
         {% endif %}
+
         <div class="sf-toolbar-info-piece">
-            Symfony <b>{{ collector.symfonyversion }}</b>
-        </div>
-        <div class="sf-toolbar-info-piece">
-            <a href="https://symfony.com/doc/{{ collector.symfonyversion }}/index.html" rel="help">Symfony Documentation</a>
+            <b>Profiler token</b>
+            <span>
+                {% if profiler_url %}
+                    <a href="{{ profiler_url }}">{{ collector.token }}</a>
+                {% else %}
+                    {{ collector.token }}
+                {% endif %}
+            </span>
         </div>
-    {% endset %}
-    {% include '@WebProfiler/Profiler/toolbar_item.html.twig' with { 'link': false } %}
 
-    {# PHP Information #}
-    {% set icon %}
-        <a href="{{ path('_profiler_phpinfo') }}">
```

Comment:
Good catch! I've readded it as a link next to the PHP version:

![phpinfo](https://cloud.githubusercontent.com/assets/73419/9005704/c242da68-3780-11e5-8429-fde31afe87e7.png)